test/e2e/run_security_labels_test.go

```diff
@@ -58,7 +58,7 @@ var _ = Describe("Podman generate kube", func() {
 
 		ctr := inspect.InspectContainerToJSON()
 		caps := strings.Join(ctr[0].EffectiveCaps, ",")
-		Expect(caps).To(Not(Equal("CAP_SYS_ADMIN")))
+		Expect(caps).To(Not(Equal("CAP_SYSADMIN")))
```

Comment:
The cap is called `SYS_ADMIN` so this looks wrong, since you match with `Not (Equal())` you turn this into a noop.